src/arrayMethodSort.js

```diff
@@ -6,7 +6,38 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
     // write code here
+    const arr = [];
+    let result;
+    let resultFunc;
+
+    if (compareFunction === undefined) {
+      resultFunc = (a, b) => String(a) > String(b) ? 1 : -1;
```

Comment:
use localeCompare to compare strings

Reply:
don't work with localeCompare)

Reply:
because the tests already have a callback function with localeCompare method
✓ custom compare function, strings: 
  ['ёж','як','аист'].sort2((a, b) => a.localeCompare(b)) (45ms)